### PR TITLE
0928 Review Adjustment

### PR DESCRIPTION
**Pull request by lyy1994**

### CODE REVIEW

File: src/helm/benchmark/static/schema.yaml [hunk 1]
```diff
@@ -2999,12 +2991,12 @@ run_groups:
...
       what: medical, open-domain, or truthful qa
       who: n/a
-      when: n/a
+      when: 2022 or before
       language: Chinese
 
   - name: cleva_summarization
     display_name: CLEVA (Chinese) summarization
-    description: "Contains one subtask so far: summarize a dialogue between a customer representative and a customer."
+    description: "Summarize a dialogue between a customer representative and a customer."
```
Comment (Jianqiao-Zhao) [inline src/helm/benchmark/static/schema.yaml:2999]: Why are we not mentioning "one subtask so far"? I have the same question for other related scenarios as well. Thanks!

Comment (Jianqiao-Zhao) [reply src/helm/benchmark/static/schema.yaml:2999]: I have no other issue with this pull request.

Comment (lyy1994) [reply src/helm/benchmark/static/schema.yaml:2999]: This is suggested by YIfan.

Comment (Jianqiao-Zhao) [reply src/helm/benchmark/static/schema.yaml:2999]: Yeah, I realize it now. Thanks.

**Review by Jianqiao-Zhao — APPROVED** [review]

**Pull request merged**